retriever/__main__.py

```diff
@@ -126,6 +127,18 @@ def main():
             return
 
         if args.command == 'ls':
+
+            if args.debugScript:
```

Comment:
Right, so I think henry asked for a debug functionality on the json scripts right before they get converted to Script objects.The ls --debug flag allows us to see what gets passed on to templates.py to be converted into a script object. I have it set so that running 
```
retriever ls --debug iris
```
returns: 
```
Values being passed to template: 
{u'citation': u'R. A. Fisher. 1936. The Use of Multiple Measurements in Taxonomic Problems. and Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.',
 u'description': u'Famous dataset from R. A. Fisher. This dataset has been corrected. Information Source: Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.',
 u'homepage': u'http://mlr.cs.umass.edu/ml/datasets/Iris',
 u'keywords': [u'plants', u'literature-compilation', u'categorical'],
 u'name': u'iris',
 u'retriever': u'True',
 u'retriever_minimum_version': u'2.0.dev',
 'tables': {u'Iris': {'cleaned_columns': False,
                      'cleanup': <retriever.lib.cleanup.Cleanup object at 0x7f3e6aa24110>,
                      'column_names_row': 1,
                      'columns': [(u'sepal_length', ('double',)),
                                  (u'sepal_width', ('double',)),
                                  (u'petal_length', ('double',)),
                                  (u'petal_width', ('double',)),
                                  (u'class', ('char', u'20'))],
                      'contains_pk': False,
                      'delimiter': ',',
                      'dialect': {u'delimiter': u',', u'header_rows': 0},
                      'fixed_width': False,
                      'format': 'tabular',
                      'header_rows': 0,
                      'missingValues': None,
                      'name': u'Iris',
                      'pk': True,
                      'record_id': 0,
                      'replace_columns': [],
                      'schema': {u'fields': [{u'name': u'sepal_length',
                                              u'type': u'double'},
                                             {u'name': u'sepal_width',
                                              u'type': u'double'},
                                             {u'name': u'petal_length',
                                              u'type': u'double'},
                                             {u'name': u'petal_width',
                                              u'type': u'double'},
                                             {u'name': u'class',
                                              u'size': u'20',
                                              u'type': u'char'}]},
                      'url': u'http://mlr.cs.umass.edu/ml/machine-learning-databases/iris/bezdekIris.data'}},
 u'title': u'Iris Plants Database',
 u'urls': {u'Iris': u'http://mlr.cs.umass.edu/ml/machine-learning-databases/iris/bezdekIris.data'},
 u'version': u'1.2.0'}
```
which is the json_object dictionary in compile.py that gets passed into our script template in templates.py.